pkg/api/controller.go

```diff
@@ -812,27 +811,35 @@ func (c *Controller) ListUsers(w http.ResponseWriter, r *http.Request, params Li
 }
 
 func normalizeAndSetToLowercase(s string) string {
```

Comment:
(Assuming this function is needed, see below:) This function is now misnamed.

Reply:
Correct, was removed all together